crates/router/src/connector/adyenplatform.rs

```diff
@@ -401,6 +398,29 @@ impl api::IncomingWebhook for Adyenplatform {
         }
     }
 
+    fn get_webhook_api_response(
+        &self,
+        request: &api::IncomingWebhookRequestDetails<'_>,
+    ) -> CustomResult<services::api::ApplicationResponse<serde_json::Value>, errors::ConnectorError>
+    {
+        if request
+            .headers
+            .contains_key(HeaderName::from_static("x-http-code"))
+        {
+            Ok(services::api::ApplicationResponse::JsonWithHeaders((
+                serde_json::json!({ "message": "accepted" }),
+                vec![(
+                    "x-http-code".to_string(),
+                    Maskable::Masked(Secret::new("404".to_string())),
+                )],
+            )))
+        } else {
+            Ok(services::api::ApplicationResponse::TextPlain(
+                "[accepted]".to_string(),
```

Comment:
There's no need to send `"[accepted]"` back in the response / any content body right?